### PR TITLE
improved 'Developer Documentation - Installation' page

### PR DESCRIPTION
**Pull request by eric-chau**
Referenced issues: 

- [backbee/backbee-standard #677](https://github.com/backbee/backbee-standard/issues/677)
- [backbee/backbee-standard #619](https://github.com/backbee/backbee-standard/issues/619)

### CODE REVIEW

File: doc/developer-documentation/installation.md [hunk 1]
```diff
@@ -22,63 +20,78 @@ Also, check your PHP version which should be 5.4 or more.
...
+### Step 4 - Site configuration
 
-You can also check the option "Create demonstration website" if you want to install a complete demonstration based on a press website, see [The MAG](http://early.backbee.com/).
+As last step, you have to choose your **site name** and fill its URL.
 
-![BackBee Installer - step 4](http://i.imgur.com/BcZ3RNA.png "BackBee Installer - step 4")
+![BackBee Installer - step 4](http://i.imgur.com/fSlenGX.png "BackBee Installer - step 4")
 
-Installation is now completed: BackBee gives you the ``Apache`` or ``Nginx`` configuration you have to set up to be able to use BackBee on the chosen domain.
+At the end of this step, BackBee will create an empty file named `INSTALL_OK` in the `public/` folder. As long as this file exists the installation cannot start. So if you want to re-run the installation process, you have to remove this file first.
```
Comment (mickaelsteinberg) [inline doc/developer-documentation/installation.md]: At the end of this step > Does this mean you have to click on the next step button to create this INSTALL_OK file? Why would I want to re-run the installation process ?

File: doc/developer-documentation/installation.md [hunk 2]
```diff
@@ -22,63 +20,78 @@ Also, check your PHP version which should be 5.4 or more.
...
+### Apache2 and Nginx virtualhosts configurations
+
+Installation is now completed. BackBee generates and provides the `Apache2` or `Nginx` virtualhost configuration. To get the best experience of BackBee CMS, we recommend you to setup the one that match your environment.
 
 ![BackBee Installer - step 5](http://i.imgur.com/T13tVjT.png "BackBee Installer - step 5")
 
 ![BackBee Installer - step 5 bis](http://i.imgur.com/ePKuLX5.png "BackBee Installer - step 5 bis")
 
-When your web server is set up, access the site by going to **[http://blogbee.dev](http://blogbee.dev)**.
+When your web server is setup, you can start using BackBee with the URL **[http://blogbee.dev](http://blogbee.dev)**.
```
Comment (mickaelsteinberg) [inline doc/developer-documentation/installation.md]: **set up** (add a space)

**Comment by mickaelsteinberg** [comment]
Other than notes on the page, :+1:

**Comment by eric-chau** [comment]
thank you @mickaelsteinberg.

**Comment by mickaelsteinberg** [comment]
Great job @eric-chau \o/

**Pull request merged**